### ISSUE TITLE
XUbuntu on Mac - Absent Enable Wifi Option

### ISSUE THREAD

**Issue by crachmad** [brittAnderson/Intro2Computing4Psychology#13]
Hi everyone! 

I've managed to have persistence on Ubuntu and managed to connect to the internet by adding the additional driver. However, after I restart my mac, the enable wifi option is gone. I'm not sure where I went wrong. Any ideas?

Christina

**Comment by seanrowe95** [comment]
@crachmad I was running into an issue earlier with connecting to wifi on my Mac, could you clarify what you mean by "the enable wifi option is gone"? (maybe post a screenshot or picture)

**Comment by crachmad** [comment]
![C681A3EF-D33C-494F-A227-4FBAA11CA220](https://user-images.githubusercontent.com/70964019/94718227-72cf7480-031f-11eb-92db-feb77fc52be0.jpg)
@seanrowe95 that's what I'm seeing. I was able to connect to the wifi the first time I installed the Broadcom but after I restart my mac, for some unknown reason the enable wifi option is not there.

**Comment by seanrowe95** [comment]
@crachmad I just tried this and ran into the same issue. I've found a work around, but it is rather annoying. In that window you have open, I clicked "do not use the device", "apply changes", which will prompt you to restart. After you restart, go back, and select the driver again (basically turning it off and on again 😂). My wifi reconnected when I did this, though it would be very annoying if we have to do this everytime we restart. Hopefully this works out, and I'll keep looking into what might be the cause behind this
